package/scripts/precook_test.py

```diff
@@ -0,0 +1,237 @@
+import os
+import time
+import random
+import gc
+import shutil
+
+pid = 1729
+
+
+def make_stat(pid, proc_pid, fixed_value, rand=False):
+    stat_fname = os.path.join(proc_pid, "stat")
+    if rand:
+        with open(stat_fname, "w") as f:
+            # Writing some pid and process name although never used
+            f.write(str(pid) + " ")  # 0 pid
+            f.write("(python3)" + " ")  # 1 Executable filename
+            # 2 to 12 are never used (as are 0 and 1) -> set to 0
+            for i in range(11):
+                f.write(str(0) + " ")
+
+            # Define upper limit for random generation of integers for utime and stime
+            time_lim = 10000000
+            # 13 utime - Time in user mode
+            f.write(str(random.randint(1, time_lim)) + " ")
+            # 14 stime - Time in kernel mode
+            f.write(str(random.randint(1, time_lim)) + " ")
+            # 15 cutime - Time of waited-for children in user mode
+            f.write(str(random.randint(1, time_lim)) + " ")
+            # 16 cstime - Time of waited-for children in kernel mode
+            f.write(str(random.randint(1, time_lim)) + " ")
```

Comment:
L22-30 are all the same code! Try something like:

```py
" ".join([str(random.randint(1, 10)) for i in range(4)])
```

for a much more concise way to write this